### ISSUE TITLE
Importing Contacts related to Accounts to allow to overwrite Contact Address

### ISSUE THREAD

**Issue by willrennie** [salesagility/SuiteCRM#214]
If you import Contacts and specify the Account to relate the Contact to, on import, the Address for the Contact is not populated from the address of the Account.

**Comment by samus-aran** [comment]
Just updating the title as standard functionality notifies the user that linking an account to a contact will  overwrite the contact's address, but viathe import you would not be able to do this.

So felt that this suggestion is to provide the check to either overwrite or not.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Importing Contacts related to Accounts to allow to overwrite Contact Address](https://trello.com/c/EbDKu6zW/155-importing-contacts-related-to-accounts-to-allow-to-overwrite-contact-address)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
